CalibFormats/HcalObjects/src/HcalText2DetIdConverter.cc

```diff
@@ -158,7 +158,17 @@ bool HcalText2DetIdConverter::init (DetId fId) {
       setField (1, calibId.rm());
       setField (2, calibId.fiber());
       setField (3, calibId.channel());
-    }
+    } else if ( calibId.calibFlavor()==HcalCalibDetId::HBX) {      
+      flavorName="HBX";
+      setField (1, calibId.ieta());
+      setField (2, calibId.iphi());
+      setField (3, calibId.depth());    
```

Comment:
Is there a reason that you are assigning a depth here (and in the HEX channels) rather than using a dummy value of -999 as in the HOX channels in L145? The depth value isn't meaningful as these channels do not read out any scintillator.